text/0000-overlapping_match.md

```diff
@@ -0,0 +1,150 @@
+- Feature Name: overlapping_match_statements
+- Start Date: 2017-09-08
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This feature is to facilitate the writing and using of `match` statements in a whole new way.
```

Comment:
"a whole new way" makes it sound like an advertisement and doesn't convey any information the reader might be interested in. It's also grammatically questionable.